### PR TITLE
Presentation tweaks

### PR DESCRIPTION
**Pull request by rlebre**
Added small tweaks to the presentation

- Dicoogle logo was a bit unformatted due to svg format: replaced to png
- list of contributors updated
- small tutorial changed to match the archive downloaded from the website
- changed index screenshot for the students to not make mistakes (some students thought that the dataset folder should be in DicoogleDir/storage because of the screenshot)
- changed architecture image
- updated maintainer email

(Note: the previous PR had to be closed due to the commits)

### CODE REVIEW

File: slides/presentation.md [hunk 1]
```diff
@@ -87,11 +84,12 @@ background-color: #cbe0f2
    - Image recognition (computer-assisted detection, content-based image retrieval, ...);
 - Knowledge extraction.
    - Obtain healthcare quality indicators.
-   - Analyses of workflow and productivity.
+   - Analyses of the workflow and productivity.
```
Comment (Enet4) [inline slides/presentation.md]: This makes it sound like we're speaking of one previously mentioned workflow. How about this?

```suggestion
   - Workflow and productivity analysis.
```

File: slides/presentation.md [hunk 2]
```diff
@@ -184,7 +181,7 @@ username: `dicoogle` <br> password: `dicoogle`
 
 - Force Dicoogle to index the set.
 
-![](img/screenshot_index.png)
+<img src="img/screenshot_index.png" width="80%" style="display:block;margin:auto"></img>
```
Comment (Enet4) [inline slides/presentation.md]:
```suggestion
<img src="img/screenshot_index.png" width="80%" style="display:block;margin:auto" />
```

File: slides/presentation.md [hunk 3]
```diff
@@ -210,7 +207,11 @@ username: `dicoogle` <br> password: `dicoogle`
 
 #### 3. Other issues?
 
-Use our [Contact Us](http://www.dicoogle.com/about) form.
+Contact the Maintainers:
+
+- Luís Bastião Silva <bastiao@bmd-software.com> (development leader)
+- Eduardo Pinho <eduardopinho@ua.pt>
```
Comment (Enet4) [inline slides/presentation.md]:
```suggestion
- Eduardo Pinho <eduardopinho@bmd-software.com>
```

File: slides/presentation.md [hunk 4]
```diff
@@ -310,36 +312,41 @@ Web UI plugins are different ([read more](https://bioinformatics-ua.github.io/di
...
-
-- Carlos Costa
-- José Luís Oliveira
-
 ]
 
 --
 
-**<i class="fa fa-github fa-lg"></i> You can be a contributor! [github.com/bioinformatics-ua/dicoogle](https://github.com/bioinformatics-ua/dicoogle)**
+**<i class="fa fa-github fa-lg"></i> You can be a contributor! [github.com/bioinformatics-ua/dicoogle](https://github.com/bioinformatics-ua/dicoogle)**
```
Comment (Enet4) [inline slides/presentation.md]: Would you like to also mention that we have a [good first issue](https://github.com/bioinformatics-ua/dicoogle/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) label?

You could probably replicate the same label style from GitHub if you copy the CSS for the `issueLabel` class.

```css
.IssueLabel {
    height: 20px;
    padding: .15em 4px;
    font-size: 12px;
    font-weight: 600;
    line-height: 15px;
    border-radius: 2px;
    box-shadow: inset 0 -1px 0 
    rgba(27,31,35,.12);
}
```

Comment (rlebre) [reply slides/presentation.md]: Nice idea! I just added an extra slide addressing this.

**Pull request merged**